tests/casper/test_message.py

```diff
@@ -17,11 +17,8 @@ def test_equality_of_copies_off_genesis(validator, empty_just):
     message = Message(None, empty_just, validator, 0, 0)
 
     shallow_copy = copy.copy(message)
```

Comment:
Does copying a message mean they are no longer equal?

Reply:
was getting crazy recursion errors failing at __hash__ on equality. Not 100% sure what was going on but copped out and removed it because we don't perform any deep copies. 

Could use another set of eyes if you want to drop the deepcopy back in to see the error.

Reply:
So I changed the validator who is creating the message to None, and this problem disappeared. This recursive error also pops up when you try and deep copy any validator, so I guess the issue is somewhere in a validator (I guess some self-referential thing) - introduced with the new genesis block.

It's not an issue for now, but I'm going to open an issue to explore more.